grammars/build.gradle

```diff
@@ -1,7 +1,7 @@
 description = "grammars for parsing ADL, ODIN and xpath"
 apply plugin: 'antlr'
 
-ext.antlrVersion = '4.9.3'
+ext.antlrVersion = '4.11.0'
```

Comment:
4.11.1 has also been released, can we update to that version directly?

Reply:
No idea why I missed that one. It is the same code, by the way, according to the release notes at https://github.com/antlr/antlr4/releases/tag/4.11.1 . Will change.

Reply:
done